src/hooks/useEscape.ts

```diff
@@ -1,8 +1,9 @@
 import React, { useEffect } from "react";
 
-const useEscape = onEscape => {
+const useEscape = (onEscape: () => void) => {
   useEffect(() => {
-    const handleEsc = event => {
+    const handleEsc = (event: KeyboardEvent) => {
+      // TS-REFACTOR-NOTE: keycode is deprecated
```

Comment:
what is the better thing to use?

Reply:
`event.code`
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
event.code === "Escape"